generators/kafka/templates/src/main/java/package/config/KafkaSseConsumer.java.ejs

```diff
@@ -1,30 +0,0 @@
-<%#
- Copyright 2013-2022 the original author or authors from the JHipster project.
-
```

Comment:
You need to remove deleted files for easier migration hehe:
https://github.com/jhipster/generator-jhipster/blob/e17f956be2f6c0062c1ce534089e7cf2e8e88bc7/generators/kafka/cleanup.mjs#L40-L43